bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAdmin.java

```diff
@@ -353,10 +345,10 @@ public void modifyTable(TableName tableName, TableDescriptor tableDescriptor) th
    if (!preserveSplits) {
       LOG.info("truncate will preserveSplits. The passed in variable is ignored.");
    }
-   DropRowRangeRequest.Builder deleteRequest = DropRowRangeRequest.newBuilder().setDeleteAllDataFromTable(true);
+   // rowKeyPrefix set to Empty ByteString to truncate data from specified table.
    return FutureUtils.toCompletableFuture(
-        bigtableTableAdminClient
-          .dropRowRangeAsync(deleteRequest.setName(toBigtableName(tableName)).build()))
+        tableAdminClientWrapper
+          .dropRowRangeAsync(tableName.getNameAsString(), ByteString.EMPTY.toStringUtf8()))
```

Comment:
In case of truncate need to call BigtableTableAdminClient#dropRowRange(tableId, rowKeyPrefix) with Empty Prefix.

Is using `ByteString.Empty` looks fine or Shall I use String `""` here?

Reply:
I think `""` would be fine